solutions/core-landing-zone/setters.yaml

```diff
@@ -103,17 +119,25 @@ data:
   # After a retention policy is locked (true), you can't delete the bucket until every log in the bucket has fulfilled the bucket's retention period
   # AU-9 PROTECTION OF AUDIT INFORMATION
   # AU-11 AUDIT RECORD RETENTION
-  #
-  # The values below must be modified to locked: true and retentionDays: 365 in a Production setting to implement above mentioned security controls.
+  # customization: The values below must be modified to locked: true and retentionDays: 365 in a Production setting to implement above mentioned security controls.
   retention-locking-policy: "false"
   retention-in-days: "1"
   #
   ##########################
   # DNS
   ##########################
   #
+  # DNS project
+  #
+  # project id for the dns project to be created, following rules and conventions
+  # customization: required
   dns-project-id: dns-project-12345
+  #
+  # Core Landing Zone fqdn. The "dns-name" must end with a "."
+  # dns-name needs needs to receive delegation from the upper level of the domain example.com.
+  # customization: required
   dns-name: "example.com."
```

Comment:
Same question as DNS Managed Zone; is this still necessary if we're deploying the DNS resources manually?

Reply:
yes, the issue you mentionned is specific to the ssc landing zone where we want dns zone to live on specific dns shard. let's leave this public package this way and create the appropriate SOP for SSC